scripts/init.js

```diff
@@ -191,4 +192,42 @@ const globalInit = {
 
 globalInit.init();
 
+// init editor if message from parent
```

Comment:
I would not prefer not to have it in the global init since it's only for authoring,
Meaning I want this code outside of the main instance of init
To avoid running or maintaining code inside the main application used only for authoring.